src/api/dashboard.js

```diff
@@ -45,11 +45,17 @@ Dashboard.prototype = {
     var initialState = this._dashboard.dashboardView.getInitialMapState();
     var currentCenter = this._dashboard.vis.map.get('center');
     var currentZoom = this._dashboard.vis.map.getZoom();
-    if (_.isEqual(currentCenter, initialState.center) && _.isEqual(currentZoom, initialState.zoom)) return {};
-    return {
-      center: currentCenter,
-      zoom: currentZoom
-    };
+    var upperLat = initialState.center[0] * 0.01;
```

Comment:
Also, we could make this a bit clearer/easier to understand by extracting a method:

```
if (this._hasStateChanged(...)) {
  return {
    center: currentCenter,
    zoom: currentZoom
  };
} else {
  return {};
}
```